L/LuaJIT/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+using BinaryBuilder
+
+name = "LuaJIT"
+# NOTE: LuaJIT has effectively moved to a "rolling release" model where users are
+# expected to track either the `v2.1` or `v2.0` branch of the Git repository rather
+# than relying on formal releases. We'll translate that to Yggdrasil versioning by
+# using the date of the commit passed to `GitSource` as the patch number with the
+# upstream version as the major and minor parts of the version. Whenever, if ever,
+# Mike Pall decides to make e.g. a v2.1.0 release, we'll have to continue to use this
+# system for it rather than reflecting the upstream version.
+version = v"2.1.20221221"
+
+# We're using the GitHub mirror because the official source seems to be acting weird
+sources = [GitSource("https://github.com/LuaJIT/LuaJIT.git",
+                     "a04480e311f93d3ceb2f92549cad3fffa38250ef")]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/LuaJIT*
+
+make -j${nproc} amalg \
+    PREFIX="${prefix}" \
+    HOST_CC="${CC_BUILD} -m${nbits}" \
+    STATIC_CC="${CC}" \
+    DYNAMIC_CC="${CC} -fPIC" \
+    CROSS="" \
+    TARGET_LD="${CC}"
+make install PREFIX="${prefix}"
+"""
+
+platforms = filter!(p -> arch(p) !== :powerpc64le, supported_platforms())
+
+products = [ExecutableProduct("luajit", :luajit),
```

Comment:
I *think* the problem on some platforms is that `luajit` is a link, not the executable itself, so you should use `luajit-2.1.0-beta3` as name of the file (sigh).

Reply:
Classic